crates/bevy_pbr/src/material.rs

```diff
@@ -180,6 +180,13 @@ pub trait Material: Asset + AsBindGroup + Clone + Sized {
     ) -> Result<(), SpecializedMeshPipelineError> {
         Ok(())
     }
+
+    /// Specify a batch key to use for shadows. Returning Some(value) allows batching of distinct materials which will produce
+    /// the same depth-prepass output (such as opaque [`StandardMaterial`]s with different textures), rendering with the first
+    /// material of the batch for all meshes.
+    fn shadow_material_key(&self) -> Option<u64> {
+        None
```

Comment:
So why is this `None` by default?

Reply:
this is in the Material trait. it's overridden in the StandardMaterial impl